lib/backup/s3remote/s3.go

```diff
@@ -61,6 +61,7 @@ func (fs *FS) Init() error {
 	}
 	configOpts := []func(*config.LoadOptions) error{
 		config.WithSharedConfigProfile(fs.ProfileName),
+		config.WithDefaultRegion("us-west-2"),
```

Comment:
this is weird. We need to provide an option to define it or change it to the most popular one
Are you sure that config will override it?

Reply:
FYI, previously the "us-east-1" region was used as default - see 9d45b46f4c53deeb37f2fa89acf37913ec066afd